moderation_queue/templates/moderation_queue/_photo-policy.html

```diff
@@ -1,3 +1,6 @@
+{% load i18n %}
+
+{% blocktrans trimmed %}
```

Comment:
I think this is too big a block to have as a single unit for translation - and it would need care to deal with because it contains HTML tags. I'd mark these up separately with `trans` or `blocktrans`:

* "Photo policy"
* "The photo must be a clear, recent, colour headshot of the candidate."
* "No text or party logos may be included in the photo, unless an unavoidable part of the background or a small rosette/badge that doesn’t interfere with the rule above. Any photo containing party text or logos should be replaced by a plain photo whenever one is available."